frontend-v2/src/features/data/LoadData.tsx

```diff
@@ -44,7 +44,7 @@ function updateDataAndResetFields(state: StepperState, data: Data) {
     const newFields = Object.keys(data[0]);
     state.setFields(newFields);
     state.setData(data);
-    const normalisedFields = newFields.map(normaliseHeader);
+    const normalisedFields = Object.fromEntries(newFields.map(normaliseHeader));
```

Comment:
A [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map) would be better here, but requires some rewriting of the rest of the code for accessing keys and values.
```suggestion
    const normalisedFields = new Map(newFields.map(normaliseHeader));
```